lib/node-labels.js

```diff
@@ -133,6 +138,8 @@ const exclusiveLabelsMap = new Map([
   // n-api is treated separately since it is not a JS core module but is still
   // considered a subsystem of sorts
   [/^doc\/api\/n-api.md$/, ['doc', 'n-api']],
+  // quic
+  [/^doc\/api\/quic.md%/, ['quic']],
```

Comment:
Should we add the `doc` label as well as `quic` when the PR is related to docs?

Refs the other definitions having `doc` as the first item in the array in addition to whatever subsystem they relate to.